lib/l10n/intl_en.arb

```
@@ -55,5 +55,351 @@
+  "selectFiatCurrencyPay": "Select the fiat currency you will pay with",
+  "selectFiatCurrencyReceive": "Select the Fiat Currency you want to receive",
+  "loadingCurrencies": "Loading currencies...",
+  "errorLoadingCurrencies": "Error loading currencies",
+  "usDollar": "US Dollar",
+  "selectCurrency": "Select Currency",
+  "searchCurrencies": "Search currencies...",
+  "noCurrenciesFound": "No currencies found",
+  
+  "@_comment_price_type_section": "Price Type Section Strings",
+  "priceType": "Price type",
+  "marketPrice": "Market price",
```

RuboCop (1.75.5)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)